docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Einstein for Developers Setup
+lang: en
+---
+
+## Set Up Overview
+
+A Salesforce admin or user with the appropriate permissions can install Einstein for Developers in a supported Salesforce org. A developer with access to the org can then:
+
+1.  Install the extension in VS Code.
+2.  Connect to an org with Einstein for Developers enabled.
+3.  Use the extension to generate Apex code from natural language instructions.
+
+### Required Salesforce Editions and User Permissions
+
+**Available in**: Developer, Enterprise, Partner Developer, Performance and Unlimited Editions.
+
+**Inoperable in**: Group, Professional, and Essentials Editions
```

Comment:
@sbudhirajadoc Can we add a footnote here saying your org must have Enhanced Domains for this feature as well? _most_ orgs should have it at this point so don't want to cause too much alarm but I'd like to call it out just in case 

Here's the enhanced domains timeline fyi - https://help.salesforce.com/s/articleView?id=sf.domain_name_enhanced_timeline.htm&type=5